common/scripted_triggers/wc_clothing_triggers.txt

```diff
@@ -295,7 +295,15 @@ religious_death_clothing_trigger = {
 	religion = { is_in_family = rf_death }
 }
 religious_gnomish_clothing_trigger = {
-	faith = { has_doctrine_parameter = religious_gnomish_clothing }
+	OR = {
+		culture = {
+			OR = {
+				has_cultural_pillar = heritage_goblin
+				has_cultural_pillar = heritage_gnomish
+			}
+		}
```

Comment:
Working with graphical look, use gfx triggers:

![image](https://user-images.githubusercontent.com/46384992/230788230-1624107d-6b18-429c-901b-e8c3798af4c0.png)

You assign gfx to culture, place gfx trigger in the code.